api/buffer/design.go

```diff
@@ -0,0 +1,549 @@
+		})
+	})
+
+	Method("ListReceivers", func() {
+		Meta("openapi:summary", "List all receivers")
+		Description("List all receivers for a given project.")
+		Result(ArrayOf(Receiver))
+		HTTP(func() {
+			GET("/receivers")
+			Meta("openapi:tag:receiver")
+			Response(StatusOK)
+		})
+	})
+
+	Method("GetReceiver", func() {
+		Meta("openapi:summary", "Get receiver")
+		Description("Get the configuration of a receiver.")
+		Result(Receiver)
+		Payload(func() {
+			receiverId()
+			Required("receiverId")
+		})
+		HTTP(func() {
+			GET("/receivers/{receiverId}")
+			Meta("openapi:tag:receiver")
+			Response(StatusOK)
+			ReceiverNotFoundError()
+		})
+	})
+
+	Method("DeleteReceiver", func() {
+		Meta("openapi:summary", "Delete receiver")
+		Description("Delete a receiver.")
+		Result(Receiver)
+		Payload(func() {
+			receiverId()
+			Required("receiverId")
+		})
+		HTTP(func() {
+			DELETE("/receivers/{receiverId}")
+			Meta("openapi:tag:receiver")
+			Response(StatusOK)
+			ReceiverNotFoundError()
+		})
+	})
+
+	Method("RefreshReceiverTokens", func() {
+		Meta("openapi:summary", "Refresh receiver tokens")
+		Description("Refresh all tokens used by a receiver.")
+		Result(Receiver)
+		Payload(func() {
+			receiverId()
+			Required("receiverId")
+		})
+		HTTP(func() {
+			POST("/receivers/{receiverId}/tokens/refresh")
+			Meta("openapi:tag:receiver")
+			Response(StatusOK)
+			ReceiverNotFoundError()
+		})
+	})
+
+	Method("CreateExport", func() {
+		Meta("openapi:summary", "Create export")
+		Description("Create a new export for an existing receiver.")
+		Result(Receiver)
+		Payload(func() {
+			receiverId()
+			Attribute("export", Export)
+			Required("receiverId", "export")
+		})
+		HTTP(func() {
+			POST("/receivers/{receiverId}/exports")
+			Meta("openapi:tag:receiver")
+			Response(StatusOK)
+			ReceiverNotFoundError()
+		})
+	})
+
+	Method("UpdateExport", func() {
+		Meta("openapi:summary", "Update export")
+		Description("Update a receiver export. The export's columns must match the existing table schema.")
+		Result(Receiver)
+		Payload(func() {
+			receiverId()
+			exportId()
+			Attribute("export", Export)
+			Required("receiverId", "export")
+		})
+		HTTP(func() {
+			PATCH("/receivers/{receiverId}/exports/{exportId}")
+			Meta("openapi:tag:receiver")
+			Response(StatusOK)
+			ReceiverNotFoundError()
+			ExportNotFoundError()
+		})
+	})
+
+	Method("DeleteExport", func() {
+		Meta("openapi:summary", "Delete export")
+		Description("Delete a receiver export.")
+		Result(Receiver)
+		Payload(func() {
+			receiverId()
+			exportId()
+			Required("receiverId", "exportId")
+		})
+		HTTP(func() {
+			DELETE("/receivers/{receiverId}/exports/{exportId}")
+			Meta("openapi:tag:receiver")
+			Response(StatusOK)
+			ReceiverNotFoundError()
+			ExportNotFoundError()
+		})
+	})
+
+	Method("Import", func() {
+		Meta("openapi:summary", "Import data")
+		Description("Upload data into the receiver. The data will be stored until import conditions are met.")
```

Comment:
`The data will be stored until import conditions are met.` where?
Nerozumiem tej vete. `Upload data into the receiver.` asi staci, zvysok bude v dokumentacii.